atomic-reactor.spec

```diff
@@ -84,24 +104,14 @@ Obsoletes:      python-dock-metadata < %{dock_obsolete_vr}
 Plugin for submitting metadata to OSBS
 
 
-%package -n python-atomic-reactor
-Summary:        Python 2 Atomic Reactor library
+%package -n python-atomic-reactor-rebuilds
+Summary:        Plugins for automated rebuilds
 Group:          Development/Tools
-License:        BSD
-Requires:       python-docker-py
-Requires:       python-requests
-Requires:       python-setuptools
-Requires:       python-dockerfile-parse >= 0.0.4
-Requires:       python-backports-lzma
-# Due to CopyBuiltImageToNFSPlugin, might be moved to subpackage later.
-Requires:       nfs-utils
-Provides:       python-dock = %{version}-%{release}
-Obsoletes:      python-dock < %{dock_obsolete_vr}
+Requires:       python-atomic-reactor = %{version}-%{release}
+Requires:       osbs
```

Comment:
This will be 'osbs >= 0.15' I guess, for `get_pod_for_build()` and `update_labels_on_build_config()` and maybe others.

But we don't have a 0.15 release yet so I'm not sure how much sense it makes to put that explicit version requirement in.